evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -20,10 +20,15 @@
                     <tr class="vote-row d-flex">
                         {% if field|is_choice_field %}
                             <td class="vote-question vote-question-choice my-auto{% if field.errors %} choice-error{% endif %}" name="{{ field.name }}">
+                            {{ field.label }}
                         {% else %}
-                            <td class="vote-question vote-question-text">
+                            <td class="vote-question vote-question-text d-flex flex-column">
+                            {{ field.label }}
+                            <div class="warning-label mt-auto d-none">
+                                <span class="fas fa-exclamation-triangle"></span>
+                                {% trans "Your answer seems meaningless. Please leave the textbox empty if you don't have any feedback for this question." %}
```

Comment:
please change to `This answer will probably be filtered out. You can simply leave the textbox empty.`

Reply:
:+1: